modules_core/styles.js

```diff
@@ -0,0 +1,34 @@
+var h = require('../h')
+var compile = require('micro-css')
+var fs = require('fs')
+var Path = require('path')
+
+// TODO distribute these styles across all
+// the relevant modules, not as a core style.
+var coreStyle = fs.readFileSync(Path.join(__dirname, '../style.css'))
+
+module.exports = {
+  needs: {
+    mcss: 'map',
+    css: 'map'
```

Comment:
I dunno, I was trying a strong stance. I reckon we could try a strong stance (maybe with the css lines commented out to start with)
Feels like a case where it might be good and bringing it back from css to mcss might be a pain